src/main/java/gyro/azure/compute/VirtualMachineResource.java

```diff
@@ -96,7 +113,9 @@
     private PublicIpAddressResource publicIpAddress;
     private String privateIpAddress;
     private String osType;
-    private DiskResource disk;
+    private DiskResource osDisk;
+    private boolean deleteOsDiskOnTerminate;
```

Comment:
This should be Boolean, just to be consistent with every other `boolean` type variables.